### DISCUSSION TITLE
About Make Readme

### DISCUSSION THREAD

**Discussion by Luis981123** [DenverCoder1/github-readme-streak-stats#650] [General]
Hello, guys.
I am newbie on github.
Could you let me know about making readme?

**Comment by Arturs1123** [comment]
Please check this link - https://streak-stats.demolab.com/demo/?user=Arturs1123&theme=github-dark-blue&hide_border=false&border_radius=4.5&locale=en&date_format=&mode=weekly&exclude_days=&sections=total%2Ccurrent%2Clongest&card_width=495&type=svg&background-type=solid&properties=fire

**Reply by Luis981123** [reply]
Thank you for your reply
You are welcome

**Reply by Arturs1123** [reply]
You are welcome

**Reply by Luis981123** [reply]
Hello

**Comment by Luis981123** [comment]
hello

**Reply by Arturs1123** [reply]
Nice to meet you

**Comment by Arturs1123** [comment]
> Hello, guys. I am newbie on github. Could you let me know about making readme?

https://www.cherryservers.com/blog/install-mongodb-ubuntu-22-04#step-1-install-mongodb-on-ubuntu-2204
Please check it!!!

**Comment by maiz-an** [comment]
### if you visit my site will be index on google so, please
https://www.mzieos.com/
